tests/pds/dataservice_test.go

```diff
@@ -35,17 +35,15 @@ var _ = Describe("{DeletePDSPods}", func() {
 				log.InfoD("Deploying DataService %v ", ds.Name)
 				isDeploymentsDeleted = false
 				dataServiceDefaultResourceTemplateID, err = pdslib.GetResourceTemplate(tenantID, ds.Name)
-				Expect(err).NotTo(HaveOccurred())
-
-				log.Infof("dataServiceDefaultResourceTemplateID %v ", dataServiceDefaultResourceTemplateID)
+				log.FailOnError(err, "Error while getting resource template")
+				log.InfoD("dataServiceDefaultResourceTemplateID %v ", dataServiceDefaultResourceTemplateID)
 
 				dataServiceDefaultAppConfigID, err = pdslib.GetAppConfTemplate(tenantID, ds.Name)
-				Expect(err).NotTo(HaveOccurred())
+				log.FailOnError(err, "Error while getting app configuration template")
 				Expect(dataServiceDefaultAppConfigID).NotTo(BeEmpty())
```

Comment:
use dash verify here

Reply:
Im checking this dataServiceDefaultAppConfigID not to be empty, not sure with dash we can verify it.